### PR TITLE
Makes SCP-173 destroyable by admins and other unprevented methods

### PR DESCRIPTION
**Pull request by gbasood**
Because holy shit dylan why did you think it would be okay to have a thing that would always exist and could not be removed by any means other than isolation

I get that it fits SCP for that but we are a spess game not an SCP game

Also added INVULNERABLE flag to the SCP for good measure

### CODE REVIEW

**Comment by 9600bauds** [comment]
this thing couldn't be destroyed by admins?
hhhhahhahaaha

File: code/WorkInProgress/Cael_Aislinn/sculpture.dm [hunk 1]
```diff
@@ -71,7 +75,7 @@
 			in_darkness = 1
 
 	//Humans can observe SCP-173. If a single human observes him, he's observed for everyone
-	//Note that humans have a 180� field of vision for the purposes of this proc
+	//Note that humans have a 180� field of vision for the purposes of this proc
```
Comment (Exxion) [inline code/WorkInProgress/Cael_Aislinn/sculpture.dm:78]: Did you just change a not broken character into a broken one?

**Comment by Exxion** [comment]
How can that diff be outdated if there has been no commit since and also the line hasn't changed?
Spoopy

**Comment by clusterfack** [comment]
Adding flags in new, tch lazy

File: code/WorkInProgress/Cael_Aislinn/sculpture.dm (hunk 1 above)
Comment (gbasood) [inline code/WorkInProgress/Cael_Aislinn/sculpture.dm:78]: Literally Atom's fault

**Comment by gbasood** [comment]
@clusterfack I tried that shit in the definition but it didn't werk

**Comment by clusterfack** [comment]
What

**Comment by gbasood** [comment]
`flags |= INVULNERABLE` did not work in the definition

If there's something else to do there I dont know what it is

**Comment by clusterfack** [comment]
Pfff obviously it wont since thats a runtime operation, take the parents flags and do | INVULNERABLE

**Comment by mrpain666** [comment]
But that's one of the best parts of SCP! The hilarity of it being so stupidly overpowered ADMINS cant even win against it.

**Comment by gbasood** [comment]
@clusterfack Can you even do that in the definition
Because if so I have no idea how
Call me a complete scrub because I am

**Comment by Shadowmech88** [comment]
@gbasood 
Take a look at one of the thousand items that have `flags = FPRINT` in their definition.

**Comment by DeityLink** [comment]
> implying you couldn't just manually set its loc to null and let the gc take care of it.

I'm sure I could find a few ways to get rid of it.

**Comment by Shadowmech88** [comment]
As it is now, the deletion of SCP-173 causes another one to spawn.
I'm pretty sure it'd trigger even if it was deleted by the garbage collector.

**Comment by gbasood** [comment]
Yeah specifically the GC would trigger it if you nullspaced it.

**Pull request merged**